src/main/kotlin/no/nav/familie/ef/sak/task/PollTekniskOpphørStatusTask.kt

```diff
@@ -0,0 +1,40 @@
+package no.nav.familie.ef.sak.task
+
+import no.nav.familie.ef.sak.service.BehandlingService
+import no.nav.familie.ef.sak.service.steg.StegService
+import no.nav.familie.prosessering.AsyncTaskStep
+import no.nav.familie.prosessering.TaskStepBeskrivelse
+import no.nav.familie.prosessering.domene.Task
+import org.springframework.stereotype.Service
+import java.time.LocalDateTime
+import java.util.*
+
+@Service
+@TaskStepBeskrivelse(taskStepType = PollStatusFraIverksettTask.TYPE,
+                     maxAntallFeil = 50,
+                     settTilManuellOppfølgning = true,
+                     triggerTidVedFeilISekunder = 15 * 60L,
+                     beskrivelse = "Sjekker status på iverksetting av teknisk opphør.")
+
+class PollTekniskOpphørStatusTask(private val stegService: StegService,
```

Comment:
Trenger vi en egen task for teknisk opphør, eller kan vi bruke `PollStatusFraIverksettTask`? 

Fordelen med å bruke en egen, er att vi kan ha ett eget steg for `VentePåTekniskOpphørStatusFraIverksett`, nå i `VentePåStatusFraIverksett` kommer den feile. Hvis vi skal ha det slik det er nå trenger vi nok å sjekke om det er teknisk opphør (og status OK) i `VentePåStatusFraIverksett` og då ikke kalle på `opprettFerdigstillOppgave`

Reply:
Jeg fjernet den och tog i bruk PollStatusFraIverksettTask  siden flyten er så pass lik...

Reply:
Fiksat det her https://github.com/navikt/familie-ef-sak/pull/713/commits/115803fb7d8e9d9da56867ba48a8a49065adf65d